staging/src/k8s.io/apimachinery/pkg/runtime/buf.go

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
nit: let's rename the file to "splice.go"

And please create some unit tests for this file.